src/main/java/codechicken/nei/NEIClientConfig.java

```diff
@@ -73,6 +73,8 @@ public class NEIClientConfig {
     public static final File heightHackHandlersFile = new File(configDir, "heighthackhandlers.cfg");
     public static final File handlerOrderingFile = new File(configDir, "handlerordering.csv");
 
+    private static Thread pluginLoader;
```

Comment:
This thread variable is no longer needed, and not keeping hold of it will free a tiny bit of memory